pontoon/base/models.py

```diff
@@ -750,7 +750,9 @@ def for_project_locale(self, project, locale, paths=None, search=None):
                 'marked': entity.marked,
                 'original_plural': entity.string_plural,
                 'marked_plural': entity.marked_plural,
-                'key': entity.key,
+                # Remove source string from the key and the delimiter
+                # as required by Translate Toolkit
+                'key': entity.key.split('\x04')[0],
```

Comment:
Clever. Almost more clever than I'd like (in that it works even if `\x04` isn't present). Almost. :P